src/course-outline/status-bar/LegacyStatusBar.test.tsx

```diff
@@ -1,15 +1,15 @@
-import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
 import { AppProvider } from '@edx/frontend-platform/react';
 import { initializeMockApp } from '@edx/frontend-platform';
 import { getConfig, setConfig } from '@edx/frontend-platform/config';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 
-import StatusBar from './StatusBar';
+import { CourseOutlineStatusBar } from '@src/course-outline/data/types';
 import messages from './messages';
 import initializeStore from '../../store';
```

Comment:
Nit
```suggestion
import initializeStore from '@src/store';
import messages from './messages';
```

There are also some paths in this file from mock that can be updated:
i.e
```
jest.mock('../../generic/data/api', () => ({
```
to
```
jest.mock('@src/generic/data/api', () => ({
```